management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java

```diff
@@ -153,6 +168,31 @@ private Supplier<Double> fromGauge(final Gauge<?> gauge) {
 
   @Override
   public void onGaugeAdded(String dropwizardName, Gauge<?> gauge) {
+
+    String serverVersion = FBUtilities.getReleaseVersionString();
+    if (serverVersion.startsWith("6.8")) {
+      // it's DSE 6.8, see if the version is low enough to require filtering
+      Matcher matcher = VERSION_PATTERN.matcher(serverVersion);
+      if (matcher.matches()) {
+        int patchVersion = Integer.parseInt(matcher.group(3));
+        if (MIN_DSE_PATCH_VERSION > patchVersion
```

Comment:
nit: this part could be precomputed in the constructor (not sure if the method is called enough times that it would make a difference)